### PR TITLE
Fix typo in wieldview

### PR DESCRIPTION
**Pull request by bell07**
causing https://github.com/minetest-mods/3d_armor/issues/55

### CODE REVIEW

**Comment by bell07** [comment]
Note: I just edited the lines in github editor without testing.

**Comment by bell07** [comment]
minetest.after() passes the third parameter "name" (line 77) as parameter to the nameless function(pname)...end call in line 72. 
Inside of function(pname) only the pname is used.
Inside of minetest.register_on_joinplayer() still "name" is used.

File: wieldview/init.lua [hunk 1]
```diff
@@ -69,12 +69,12 @@ end
 minetest.register_on_joinplayer(function(player)
 	local name = player:get_player_name()
 	wieldview.wielded_item[name] = ""
```
[the remark this answers is not in the document]
Comment (bell07) [reply wieldview/init.lua:71]: minetest.after() passes the third parameter "name" (line 77) as parameter to the nameless function(pname)...end call in line 72. 
Inside of function(pname) only the pname is used.
Inside of minetest.register_on_joinplayer() still "name" is used.

**Comment by BuckarooBanzay** [comment]
the additional `pname` if kinda superfluous, using the `name` from the parent scope would have been ok too, but yeah: looks good :+1:

**Pull request merged**